index.html

```diff
@@ -895,34 +727,20 @@ <h3 class="bannerbox__title">Newsletter</h3>
             });
 
             // Submit Function for needRide (Rider) form.
-            $('form[name="needRide"]').submit(function() {
+            $('form[name="needRide"]').on('submit', function() {
                 document['needRide'].action = "https://api.carpoolvote.com/v1.1/rider";
```

Comment:
Is these action attribute updates in the JS necessary? They're already being set in the HTML.